conanfile.py

```diff
@@ -93,6 +95,7 @@ def package_info(self):
             "fmt::fmt",
             "flatbuffers::flatbuffers",
             "ghc-filesystem::ghc-filesystem",
+            "mpark-variant/1.4.0",
```

Comment:
Shouldn't this be the alias to the CMake target?
```suggestion
            "mpark_variant::mpark_variant",
```